CorpusSearch/Dockerfile

```diff
@@ -0,0 +1,29 @@
+﻿FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
```

Comment:
I presume we're on 6 so we can merge this PR, get Docker running, then make the jump to 8?

Reply:
Yes. I can go ahead and merge.